data-prepper-expression/src/test/java/org/opensearch/dataprepper/expression/ParseTreeCoercionServiceTest.java

```diff
@@ -350,11 +323,9 @@ private static Stream<Arguments> provideSupportedJsonPointerValues() {
                 Arguments.of("test value"),
                 Arguments.of(1.1f),
                 Arguments.of(1.1),
+                Arguments.of(List.of("test_value")),
+                Arguments.of(Map.of("test_key", "test_value")),
                 Arguments.of((Object) null)
         );
     }
-
-    private static Stream<Arguments> provideUnSupportedJsonPointerValues() {
-        return Stream.of(Arguments.of(new HashMap<>()));
```

Comment:
Instead of `new HashMap<>()` could you provide some other class? Perhaps `new AtomicBoolean()` or `new JsonFactory()`.

Reply:
I'll try those! Thanks I tried some other classes but nothing was failing

Reply:
Thanks for trying. I do see that these are integration tests. Can you find the corresponding unit tests and use mocking to keep the code coverage?